transformer_engine/paddle/layer/transformer.py

```diff
@@ -154,6 +157,9 @@ def __init__(
         self.tensor_parallel = self.tp_size > 1
         self.sequence_parallel = self.tensor_parallel and sequence_parallel
         self.hidden_dropout_rng_state_name = hidden_dropout_rng_state_name
+        self.use_cudagraph = use_cudagraph
```

Comment:
I don't think the `use_cudagraph` attribute is used anywhere:
```suggestion
```

Reply:
Removed.